src/Catty/ViewController/Help/HelpWebViewController.swift

```diff
@@ -43,14 +46,29 @@ class HelpWebViewController: UIViewController, UIWebViewDelegate, UIScrollViewDe
 
     deinit {
         url = nil
-        webView.delegate = nil
+        webView.uiDelegate = nil
         webView = nil
     }
 
     // MARK: - View Lifecycle
 
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        let webConfiguration = WKWebViewConfiguration()
+        webConfiguration.allowsInlineMediaPlayback = true
+
+        webView = WKWebView(frame: self.viewForWebView.bounds, configuration: webConfiguration)
+        webView.uiDelegate = self
+        webView.navigationDelegate = self
+        webView.translatesAutoresizingMaskIntoConstraints = false
+        viewForWebView.addSubview(webView)
+
+        viewForWebView.addConstraint(NSLayoutConstraint(item: webView!, attribute: .leading, relatedBy: .equal, toItem: viewForWebView, attribute: .leading, multiplier: 1.0, constant: 0.0))
```

Comment:
Please remove force unwrapping here (this will be reported by our linter in the future).